spark-gremlin/src/test/java/org/apache/tinkerpop/gremlin/spark/structure/SparkTest.java

```diff
@@ -51,6 +51,15 @@
  * @author Marko A. Rodriguez (http://markorodriguez.com)
  */
 public class SparkTest extends AbstractSparkTest {
+    @Test
+    public void testCustomizedSparkAppName() {
+	final String appName = "SparkAppNameTest";
+	final org.apache.spark.SparkConf sparkConfiguration = new org.apache.spark.SparkConf();
```

Comment:
Just curious, why did you use the fully qualified name here instead of importing?

Reply:
No special purpose. My impression is many "import" sometimes brings trouble and confusion to the current file namespace, like the org.apache.commons.configuration2.Configuration and org.apache.commons.configuration.Configuration. So, I tried to avoid that. In addition, org.apache.spark.SparkConf is not too long and not impact the reading.